modules/monitoring/application/views/scripts/health/info.phtml

```diff
@@ -56,6 +56,12 @@ if (! $this->compact): ?>
                             ? $this->programStatus->endpoint_name
                             : $this->translate('N/A') ?></td>
                 </tr>
+                <tr>
+                    <th><?= $this->translate('Active IcingaWeb Endpoint'); ?></th>
+                    <td><?= gethostname()
+                            ? gethostname()
+                            : $this->translate('N/A') ?></td>
+                    </tr>
```

Comment:
* Please ensure your indentation is correct
* `IcingaWeb` should read `Icinga Web 2`
* If the value you're comparing in a ternary operation is about to be used as truthy result anyway you can shorten the statement by just using `?:`

```php
                <tr>
                    <th><?= $this->translate('Active Icinga Web 2 Endpoint'); ?></th>
                    <td><?= gethostname() ?: $this->translate('N/A') ?></td>
                </tr>
```

Reply:
Thanks @nilmerg 
I fixed with your suggestion and checked indentation.
Please see updated code.